Extension/src/LanguageServer/client.ts

```diff
@@ -883,6 +886,32 @@ export class DefaultClient implements Client {
         });
     }
 
+    public async showSelectCompiler(paths: string[]): Promise<number> {
+        const options: vscode.QuickPickOptions = {};
+        options.placeHolder = localize("select.compile.commands", "Select a compiler to configure for IntelliSense");
+
+        const items: IndexableQuickPickItem[] = [];
+        for (let i: number = 0; i < paths.length; i++) {
+            let option: string | undefined;
+            let isCompiler: boolean = false;
+            if (paths[i].indexOf("\\") > 0) {
```

Comment:
As mentioned in chat, I don't think this path will get resolved on Linux/Mac...not sure if that should block the PR or not.